src/Mod/Start/Gui/StartView.cpp

```diff
@@ -111,73 +113,90 @@ StartView::StartView(QWidget* parent)
     setObjectName(QLatin1String("StartView"));
     auto hGrp = App::GetApplication().GetParameterGroupByPath(
         "User parameter:BaseApp/Preferences/Mod/Start");
-    auto cardSpacing = hGrp->GetInt("FileCardSpacing", 30);  // NOLINT
-
-    auto scrolledWidget = gsl::owner<QWidget*>(new QWidget(this));
-    _contents->setWidget(scrolledWidget);
-    _contents->setHorizontalScrollBarPolicy(Qt::ScrollBarPolicy::ScrollBarAsNeeded);
-    _contents->setVerticalScrollBarPolicy(Qt::ScrollBarPolicy::ScrollBarAsNeeded);
-    _contents->setWidgetResizable(true);
-    auto layout = gsl::owner<QVBoxLayout*>(new QVBoxLayout(scrolledWidget));
-    layout->setSizeConstraint(QLayout::SizeConstraint::SetMinAndMaxSize);
-
-    auto firstStart = hGrp->GetBool("FirstStart2024", true);  // NOLINT
-    if (firstStart) {
-        auto firstStartRegion = gsl::owner<QHBoxLayout*>(new QHBoxLayout);
-        firstStartRegion->addStretch();
-        auto firstStartWidget = gsl::owner<FirstStartWidget*>(new FirstStartWidget(this));
-        firstStartRegion->addWidget(firstStartWidget);
-        firstStartRegion->addStretch();
-        layout->addLayout(firstStartRegion);
-
-        // Try to further differentiate the checkbox below, when the First Start box is shown
-        auto line = new QFrame();
-        line->setFrameShape(QFrame::HLine);
-        line->setFrameShadow(QFrame::Sunken);
-        layout->addWidget(line);
-    }
-
-    // Launch start automatically?
-    _showOnStartupCheckBox = gsl::owner<QCheckBox*>(new QCheckBox());
-    bool showOnStartup = hGrp->GetBool("ShowOnStartup", true);
-    _showOnStartupCheckBox->setCheckState(showOnStartup ? Qt::CheckState::Unchecked
-                                                        : Qt::CheckState::Checked);
-    connect(_showOnStartupCheckBox, &QCheckBox::toggled, this, &StartView::showOnStartupChanged);
-    layout->addWidget(_showOnStartupCheckBox);
+    auto cardSpacing = hGrp->GetInt("FileCardSpacing", 15);  // NOLINT
+
+    // First start page
+    auto firstStartScrollArea = gsl::owner<QScrollArea*>(new QScrollArea());
+    auto firstStartRegion = gsl::owner<QHBoxLayout*>(new QHBoxLayout(firstStartScrollArea));
+    firstStartRegion->addStretch();
+    auto firstStartWidget = gsl::owner<FirstStartWidget*>(new FirstStartWidget(this));
+    connect(firstStartWidget->doneButton,
+            &QPushButton::clicked,
+            this,
+            &StartView::firstStartWidgetDoneClicked);

```

Comment:
This is breaking encapsulation and introducing a hard dependency that we should avoid. Instead, modify the firstStartWidget to emit its own signal when it is closed (via whatever mechanism -- external code doesn't need to know that), and connect to that signal.

Reply:
I've tried to make the FirstStartWidget emit it's own signal but for some reason (really no idea what was going on) the start page just doesn't render with no errors or warnings when I do that.

Basically what ı attempted was:

FirstStartWidget.h
```cpp
public:
    void doneClicked();
```

FirstStartWidget.cpp
```cpp
_doneButton = gsl::owner<QPushButton*>(new QPushButton);
connect(_doneButton, &QPushButton::clicked, this, &FirstStartWidget::doneClicked);
```

StartView.cpp
```cpp
auto firstStartWidget = gsl::owner<FirstStartWidget*>(new FirstStartWidget(this));
connect(firstStartWidget,
        &FirstStartWidget::doneClicked,
        this,
        &StartView::firstStartWidgetDoneClicked);
```

I'm not that experienced with Qt widgets signals/slots so I'm not sure what's going on, would appreciate guidance here.

Reply:
Sure: after the meeting today I'll give it a look and give you a commit to add. (My first suggestion is don't call the signal `doneClicked` -- we don't care about that internal detail of the widget, so call it something like `closed` or `dismissed`).

Reply:
I think I got it. You can review the latest commit.